### ISSUE TITLE
race and ethnicity fields

### ISSUE THREAD

**Issue by mcfrank** [langcog/web-cdi#358]
We currently have two fields - `child ethnicity` and `child hispanic latino`. These are labeled incorrectly and need to be swapped to `child race` and `child ethnicity` (which means hispanic/latino). Also the "no ethnicity" answer should be corrected to "no race".

**Comment by HenryMehta** [comment]
@mcfrank 
I've updated the field names for English.  For other languages the Child's Ethnicity translation will automatically move across and be translated as previously.  If these languages had previously translated Child's Ethnicity and Child's Race, this means that they'll now be wrong.

I am happy to review the French, Dutch and Spanish and ensure they are right.  I need translations for Korean and Hebrew.  I think I have emails for the two main contacts so I'm happy to make contact if you want

**Comment by HenryMehta** [comment]
I have now worked through all instruments and I think the translations are good across all languages.  Best check would be for native speakers to confirm within development

**Comment by HenryMehta** [comment]
@vmarchman yes, you should.  But I did this in August so I'm not 100% certain what I did now.
